pkg/api/validate/validators.go

```diff
@@ -278,13 +119,140 @@ func (v *Validator) validateUpdateContainerService(cs, oldCs *OpenShiftManagedCl
 	return
 }
 
-func (v *Validator) validateProperties(p *Properties, location string, externalOnly bool) (errs []error) {
+func validateFQDN(p *api.Properties, location string) (errs []error) {
```

Comment:
note: the reordering of functions as part of this PR makes reviewing harder - separating the reordering to an otherwise no-op commit would have been handy, so that commit could be excluded at review time

Reply:
rollbacked where I could